src/Cache/RedisDistributedCache.cs

```diff
@@ -272,7 +294,7 @@ public void Remove(string key)
         /// <returns>Returns the task to execute.</returns>
         public Task RemoveAsync(string key, CancellationToken token = default)
         {
-            return Task.Run(() => this.Remove(key), token);
+            return Task.Run(() => Remove(key), token);
```

Comment:
Unless the method is static, StyleCop rule is to include this.

Reply:
FIxed all StyleCop rules, except of _type in word Redis_